paasta_tools/kubernetes_tools.py

```diff
@@ -1912,7 +1913,11 @@ def format_pod_event_messages(
     rows: List[str] = list()
     rows.append(PaastaColors.blue(f"Pod Events for {pod_name}"))
     for message in pod_event_messages:
-        rows.append(f"   Event at {message['timeStamp']}: {message['message']}")
```

Comment:
no need to catch exception here, you should be able to use .get:

    rows.append(f"   Event at {message.get('timeStamp', 'unknown time')}: {message['message']}")

Reply:
thanks, didn't know about that.